urumanifest/github.py

```diff
@@ -383,10 +389,19 @@ def _unpack_artifact(staging_path: Path, database: _WorkflowDatabase, rev: str,
         output_path.mkdir(parents=True, exist_ok=True)
 
         def iter_client_dir():
-            for i in filter(lambda x: not x.is_dir(), archive.infolist()):
+            for i in archive.infolist():
                 member_path = PurePosixPath(i.filename)
-                if len(member_path.parts) == 2 and member_path.parts[0] == "client":
-                    yield i
+                client_folder_name = "client"
+                if len(member_path.parts) >=  2 and member_path.parts[0] == client_folder_name:
+                    is_mac_app_bundle = member_path.parts[1].endswith(".app")
+                    if not i.is_dir() and len(member_path.parts) ==  2:
+                        yield ArtifactInfo(path=member_path.name, zipinfo=i, name=member_path.name)
+                    elif is_mac_app_bundle and not i.is_dir():
+                        # remove "client/"
+                        path = member_path.relative_to(client_folder_name)
+                        yield ArtifactInfo(path=path, zipinfo=i, name=member_path.parts[1])
```

Comment:
What is going on here?

Reply:
This is constructing paths to the bundle, but not the files in the bundle. For example:
`client/plClient.app/Contents/Info.plist`
Should only be added to the control json as:
`plClient.app`